### ISSUE TITLE
JAWS 2018/19 does not read aria-live region in MS Edge

### ISSUE THREAD

**Issue by Mirodil** [FreedomScientific/VFO-standards-support#192]
### Summary

> JAWS 2018/19 does not read aria-live region in MS Edge. JAWS works as expected on IE 11, Chrome, FireFox.
> 
> Example:
> 1. Go to https://support.freedomscientific.com/Training/Surfs-Up/AriaLiveRegions.htm using MS Edge
> 2. And notice JAWS does not read aria-live region

### Expected result

> Expected: JAWS would read aria-live region when it changes

### Actual result

> JAWS does not read aria-live region

### Example

> https://support.freedomscientific.com/Training/Surfs-Up/AriaLiveRegions.htm

#### JAWS version and build number
- 2018 (build 1710.42 ILM)
- 2019.1903.47 ILM

 
#### Operating System and version
- Windows 10

#### Browser and version:
- Microsoft Edge 38.14393.0.0
- Microsoft Edge 42.17134.1.0

**Comment by stevefaulkner** [comment]
@Mirodil Since Edge is moving to using the chrome rendering engine we are no longer putting time into current Edge related bugs. you can download the [new edge browser](https://www.microsoftedgeinsider.com/en-us/) to try for yourself. I tested with the edge dev version and found that JAWS did not announce the live region content (NVDA did). Closing this for now, will re-open if the issue still exists once the public release version of Edge is available.
